packets.go

```diff
@@ -43,13 +43,18 @@ type EncryptionHeader struct {
 
 // encryptionBlock contains a block of encrypted data. It contains
 // the ciphertext, and any necessary authentication Tags.
-type encryptionBlock struct {
+type encryptionBlockV1 struct {
 	_struct            bool                   `codec:",toarray"`
 	HashAuthenticators []payloadAuthenticator `codec:"authenticators"`
 	PayloadCiphertext  []byte                 `codec:"ctext"`
 	seqno              packetSeqno
 }
 
+type encryptionBlockV2 struct {
```

Comment:
Turns out `omitempty` and `toArray` don't work together, so I had to split up the blocks. The ugorji/codec docs are unclear, so I might file an issue about it with them.